app/controllers/FileUploadController.scala

```diff
@@ -76,7 +76,13 @@ class FileUploadController @Inject() (
             )).mapTo[FileUploadState].flatMap { _ =>
               Future.successful(Redirect(routes.FileUploadController.showFileUpload()))
             }
-          case _ => Future.successful(fileStateErrror)
+          case _ =>
```

Comment:
on the other two usages in this controller you match Some and None.
It is upto you if you want to make it consistant

Reply:
Sorry, was already like that, not introduced here.

Reply:
Good spot.  Sometimes `case _` is safer just in case someone adds an "if" to the `case Some(x)` statement and you then have a hole to fall down.

But as you say, 'twas like that already.